README.md

```diff
@@ -1,38 +1,145 @@
-# Filter Employee Modal
+# Zadanie Rekrutacyjne - Michał Skrzypczyński
 
-## Description
+Plusy:
+- Wersja live
+- Readme
+- Fajne użycie styled-components z propsami
+- Proptypes
+- Custom hooks, np. z wielkością okna
 
-A simple website that filters employees by filters selected by a user.
-Made for recruitment purposes.
 
-## Preview
+Minusy:
+- config z pracownikami w App.js ?
+- Zero testów
+- Zakomentowany kod - np w useWindowsSize.js
+- Komponent funkcyjny Home, w którym dużo się dzieje, bardzo dużo hooków, zasadne by tu było użycie klasy i wywalenie większości metod na zewnątrz. Tutaj mocno widać brak Reduxa czym skutkuje.
```

Comment:
Zastanawiałem się, czy nie wykorzystać Reduxa, teraz żałuję, że tego nie zrobiłem ;)

Reply:
Może trochę się zbytnio czepiam -> bo to w końcu tylko jeden modal, ale mamy np. aplikację w które jest ponad 500 komponentów. Ona nas nauczyła, że Redux jest super przydatny, pomaga znacząco odchudzic komponenty jak i przepływ propsów. Jak się do tego doda narzędzie Immer to ma się pancerny state manager

Reply:
Plus komponenty funkcyjne z taką ilością logiki i metodami są za każdym renderem na nowo definiowane. Polecam artykuły na stack overflow w tematyce 'methods in stateless/functional components'